buildenv/jenkins/common/build.groovy

```diff
@@ -528,7 +528,10 @@ def archive_diagnostics(javaVersionJdkImageDir = null) {
     findCrashDataCmd += " -o -name 'jitdump.*.dmp'"
     findCrashDataCmd += " -o -name 'Snap.*.trc'"
 
-    if (SPEC.contains('win')) {
+    if (SPEC.contains('linux')) {
+        // collect *.debuginfo files to help diagnose omr_ddrgen failures
+        findCrashDataCmd += " -o -name '*.debuginfo'"
```

Comment:
Are you going to need anything else like .so files?

Reply:
No, `omr_ddrgen` only reads `.debuginfo` files (on Linux).